server/api/trend.go

```diff
@@ -152,9 +152,9 @@ func (h *trendHandler) getStoreFlow(stats statistics.StoreHotPeersStat, regionSt
 		return
 	}
 	if stat, ok := stats[storeID]; ok {
-		storeFlow = stat.TotalLoads[regionStatKind]
+		storeFlow = stat.TotalBytesRate
```

Comment:
The variable name doesn't match its meaning.

Reply:
only revert it https://github.com/tikv/pd/pull/3591/files#diff-e7c11b02e9457f3fd65407917fc0c632537ea493dd5499aa18dfc8bc2c067d52L156

Reply:
So we need to make it better.

Reply:
done